module/reader/src/Application/Controller/Api/ReaderController.php

```diff
@@ -224,8 +219,9 @@ public function createReader(Request $request): Response
         if ($name && $type) {
             $command = new CreateReaderCommand($name, $type);
             $this->messageBus->dispatch($command);
-            $response = $this->createRestResponse(['id' => $command->getId()->getValue()], [], Response::HTTP_CREATED);
+            $response = new CreatedResponse($command->getId());
         } else {
+            // @todo ??????? error ?????
```

Comment:
todo what ??

Reply:
We just respond with "error" message. This means nothing :) What should i set?